tests/foreman/api/test_errata.py

```diff
@@ -112,7 +114,7 @@ def _install_package(self, clients, host_ids, package_name, via_ssh=True):
                 'content_type': 'package',
                 'content': [package_name],
             })
-            self._validate_package_installed(clients, package_name)
+            self._validate_package_installed(clients, rpm_package_name)
```

Comment:
You should probably use smth like `rpm_package_name if rpm_package_name else package_name` not to introduce regressions.

Reply:
in case of ssh install we need to use package_name, in case of REX  we need both

Reply:
@abalakh adding that param we introduce no regression , as all other tests use ssh

Reply:
My point was - existing tests don't use `rpm_package_name` kwarg, so it'll be `None`. If you leave this check as it is - tests will ensure package `None` is installed. Or am i missing something?